modules/nlu-testing/src/bot-templates/bp-nlu-slot-extraction/entities/gun.json

```diff
@@ -0,0 +1,27 @@
+{
+  "id": "gun",
+  "name": "gun",
+  "type": "pattern",
+  "occurrences": [],
+  "sensitive": true,
+  "fuzzy": 0.8,
+  "examples": [
+    "A1",
+    "A12",
+    "AB1",
+    "AB12",
+    "a1",
+    "A12",
+    "Ab1",
+    "aB12",
+    "A-1",
+    "A-12",
+    "AB-1",
+    "AB-12",
+    "a-1",
+    "A-12",
+    "Ab-1",
+    "aB-12"
+  ],
+  "pattern": "[A-z]{1,2}-?[0-9]{1,2}"
```

Comment:
I don't know if this is on purpose but gun and flight number will clash as patterns are quite similar

Reply:
It's on purpose. 

For guns only it's one intent in one context so it cannot fail, same for flight number but we wanted to try to make it clash as a difficult test to improve later.